java/test/jmri/util/prefs/JmriPreferencesProviderTest.java

```diff
@@ -108,11 +108,9 @@ public void testIsFirstUse(TestInfo info, @TempDir File folder) throws IOExcepti
     @Test
     @SuppressWarnings("deprecated") // Package.getPackage()
```

Comment:
I suppose this line can be removed now when `Package.getPackage("jmri.util")` isn't used anymore.